test/4.opts.test.js

```diff
@@ -19,7 +20,11 @@ nock('http://dev.com')
 describe('fast-proxy smoke', () => {
   it('init', async () => {
     const fastProxy = require('../index')({
-      base: 'http://127.0.0.1:3000'
+      base: 'http://127.0.0.1:3000',
+      queryString: {
+        stringify: (value) => querystring.stringify(value),
+        parse: (value) => querystring.parse(value)
+      }
```

Comment:
```suggestion
      queryString
```

Reply:
I just pushed the changes. Thank you for the review!